app/src/main/res/layout/connection_details_website_info.xml

```diff
@@ -0,0 +1,109 @@
+                android:importantForAccessibility="no"
+                android:scaleType="fitCenter"
+                android:layout_marginStart="16dp"
+                android:layout_gravity="center_vertical"
+                tools:drawable="@drawable/ic_internet" />
+
+            <TextView
+                android:id="@+id/title"
+                style="@style/QuickSettingsText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:paddingTop="8dp"
+                android:paddingStart="8dp"
+                android:textStyle="bold"
+                tools:text="Wikipedia"
+                tools:ignore="RtlSymmetry" />
+        </LinearLayout>
+
+        <TextView
+            android:id="@+id/url"
+            style="@style/QuickSettingsText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingTop="8dp"
+            tools:text="https://wikipedia.org" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:minHeight="@dimen/tracking_protection_item_height"
+            android:orientation="horizontal"
+            style="@style/QuickSettingsText"
+            android:paddingTop="8dp"
+            android:paddingBottom="8dp">
+            <ImageView
+                android:id="@+id/securityInfoIcon"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:importantForAccessibility="no"
+                app:srcCompat="@drawable/mozac_ic_lock"
+                android:paddingStart="0dp"
+                android:paddingEnd="8dp"/>
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+                <TextView
+                    android:id="@+id/securityInfo"
+                    style="@style/QuickSettingsLargeText"
+                    android:layout_width="match_parent"
+                    android:paddingStart="0dp"
+                    android:layout_height="wrap_content"
+                    android:paddingEnd="0dp"
+                    tools:text="Connection is secure" />
+                <TextView
+                    android:id="@+id/certificateInfo"
+                    style="@style/QuickSettingsSmallText"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    android:paddingTop="0dp"
+                    android:paddingBottom="2dp"
+                    tools:text="Verified By: E-Corp" />
+            </LinearLayout>
```

Comment:
Let's add some new lines to space things out a bit. This is to align with the spacing you already utilize in this file above.

```suggestion

            <ImageView
                android:id="@+id/securityInfoIcon"
                android:layout_width="wrap_content"
                android:layout_height="wrap_content"
                android:importantForAccessibility="no"
                app:srcCompat="@drawable/mozac_ic_lock"
                android:paddingStart="0dp"
                android:paddingEnd="8dp"/>

            <LinearLayout
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:orientation="vertical">

                <TextView
                    android:id="@+id/securityInfo"
                    style="@style/QuickSettingsLargeText"
                    android:layout_width="match_parent"
                    android:paddingStart="0dp"
                    android:layout_height="wrap_content"
                    android:paddingEnd="0dp"
                    tools:text="Connection is secure" />

                <TextView
                    android:id="@+id/certificateInfo"
                    style="@style/QuickSettingsSmallText"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    app:layout_constraintEnd_toEndOf="parent"
                    app:layout_constraintStart_toStartOf="parent"
                    app:layout_constraintTop_toTopOf="parent"
                    android:paddingTop="0dp"
                    android:paddingBottom="2dp"
                    tools:text="Verified By: E-Corp" />
            </LinearLayout>
```